### PR TITLE
add location routes and service

### LINKED ISSUE #16: location routes and dataService

### PR DESCRIPTION
**Pull request by kc-leung**
fix #16 

@coreyjjames could you explain me on Thursday how to do the tester?

### CODE REVIEW

**Comment by coreyjjames** [comment]
> fix #16
> 
> @coreyjjames could you explain me on Thursday how to do the tester?

Sure I can go through the tests Thursday. It kind of looks complicated, but it is simple when you know what to do. I am using the same Tests for all the Routes, so it's mostly copying and pasting. I am just checking that each Route can do a simple CRUD operation and I am currently not testing edge cases.

File: routes/locations/locations.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const express = require("express");
+const router = express.Router();
+const dataService = require("./locationsService");
+// const dataService = require("../../dataService");
+const data = dataService();
+
+// get list of restaurants from db
```
Comment (coreyjjames) [inline routes/locations/locations.js]: The comment says restaurants instead of Location

**Review by coreyjjames — CHANGES_REQUESTED** [review]
Looks good just found one typo and when we work on the tests for this Route, we can confirm the Routes work.

**Review by coreyjjames — APPROVED** [review]
Looks good
